### PR TITLE
Removes screening smoke tests

### PR DESCRIPTION
**Pull request by edgibbs**

### CODE REVIEW

**Comment by TylerOSI** [comment]
Jenkins test this please

**Comment by TylerOSI** [comment]
Jenkins test this please

**Comment by TylerOSI** [comment]
Jenkins please test this

**Comment by TylerOSI** [comment]
Jenkins please test this

**Comment by TylerOSI** [comment]
Jenkins test this please

**Comment by TylerOSI** [comment]
Jenkins test this please

File: spec/smoke/intake_snapshot_screening_spec.rb [hunk 1]
```diff
@@ -1,17 +1,6 @@
...
-  end
-
-  it 'when clicking Start Screening link' do
-    click_button('Start Screening')
-    expect(page).to have_field('Title/Name of Screening')
-  end
-end
-
-describe 'Snapshot Page Ready', type: :feature do
+describe 'Snapshot Page Ready 2', type: :feature do
```
Comment (pokharelmanish) [inline spec/smoke/intake_snapshot_screening_spec.rb]: same

File: spec/smoke/intake_dashboard_ready_spec.rb [hunk 2]
```diff
@@ -1,12 +1,11 @@
 # frozen_string_literal: true
 
-describe 'Dashboard Ready', type: :feature do
+describe 'Dashboard Ready 2', type: :feature do
```
Comment (pokharelmanish) [inline spec/smoke/intake_dashboard_ready_spec.rb]: `Dashboard Ready 2` why 2?

**Comment by TylerOSI** [comment]
not sure. @edgibbs ?

File: spec/smoke/intake_dashboard_ready_spec.rb (hunk 2 above)
Comment (edgibbs) [reply spec/smoke/intake_dashboard_ready_spec.rb]: Yeah, I need to clean those up, good catch. Jenkins was running specs that didn't exist on this branch, so I added these in the descriptions so we could rule out that it was really running this branch.  It turned out the docker-compose file was hardcoded to check out the master branch.

**Pull request merged**